layouts/_default/baseof.html

```diff
@@ -108,6 +108,10 @@
 
     {{ template "_internal/google_analytics.html" . }}
 
+    {{ if .Site.Params.fathomAnalytics.siteID }}
```

Comment:
Almost there.

We still need to check for `fathomAnalytics` or we will get warning when not using Fathom.
```suggestion
    {{ if and .Site.Params.fathomAnalytics .Site.Params.fathomAnalytics.siteID }}
```